src/core/modules/Planet.cpp

```diff
@@ -639,8 +639,9 @@ QString Planet::getInfoString(const StelCore* core, const InfoStringGroup& flags
 				if (englishName!="Sun")
 					oss << QString("%1: %2").arg(q_("Mean solar day"), StelUtils::hoursToHmsStr(qAbs(getMeanSolarDay()*24))) << "<br />";
 			}
-			else if (re.period==0.f)
+			else if (re.period==0.f) // WILL NEVER BE CALLED! In this case siderealPeriod==0, the enclosing brace will exclude this!
```

Comment:
Why? See: https://en.wikipedia.org/wiki/Chaotic_rotation

Or you removed all `rot_periode=0` from ssystem_major.ini file?

Reply:
Oops, you are right. Probably these lines are from a late night in 2017. 
rot_periode since then defaults to the orbit_period. I must have disabled a few too many lines... I will reactivate the chaotic moons. *-)